pkg/app/mutate_app_psp_removal.go

```diff
@@ -0,0 +1,150 @@
+package app
+
+import (
+	"context"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	"github.com/giantswarm/app/v7/pkg/key"
+	"github.com/giantswarm/k8smetadata/pkg/label"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	capiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/app-admission-controller/pkg/mutator"
+	"github.com/giantswarm/app-admission-controller/pkg/project"
+)
+
+var (
+	// pssCutoffVersion represents first & lowest Giant Swarm Release version
+	// which does not support PodSecurityPolicies.
+	pssCutoffVersion, _ = semver.NewVersion("v19.2.0")
```

Comment:
Fun stuff - this changed in the last couple days :sweat_smile: 
```suggestion
	pssCutoffVersion, _ = semver.NewVersion("v19.3.0")
```